### PR TITLE
[Console] Fix deprecation when description is null

### PR DESCRIPTION
**Pull request by HypeMC**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

Fixes `str_replace(): Passing null to parameter #3 ($subject) of type array|string is deprecated` when `getDefaultDescription()` returns `null`, caused by #46574.

### CODE REVIEW

File: src/Symfony/Component/Console/DependencyInjection/AddConsoleCommandPass.php [hunk 1]
```diff
@@ -56,6 +56,7 @@ public function process(ContainerBuilder $container)
         foreach ($commandServices as $id => $tags) {
             $definition = $container->getDefinition($id);
             $definition->addTag($this->noPreloadTag);
+            /** @var class-string<Command> $class */
```
Comment (derrabus) [inline src/Symfony/Component/Console/DependencyInjection/AddConsoleCommandPass.php]: This change looks unrelated and should be reverted.

**Review by derrabus — APPROVED** [review]

File: src/Symfony/Component/Console/DependencyInjection/AddConsoleCommandPass.php (hunk 1 above)
Comment (HypeMC) [reply src/Symfony/Component/Console/DependencyInjection/AddConsoleCommandPass.php]: Done

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Good catch, thanks @HypeMC.
